apis/v1alpha2/backendtlspolicy_types.go

```diff
@@ -0,0 +1,156 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/gateway-api/apis/v1beta1"
+)
+
+// +genclient
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:storageversion
+// +kubebuilder:resource:categories=gateway-api,shortName=btlspolicy
+// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
+
+// BackendTLSPolicy provides a way to publish TLS configuration
+// that enables a gateway client to connect to a backend pod.
+type BackendTLSPolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the desired state of BackendTLSPolicy.
+	Spec BackendTLSPolicySpec `json:"spec"`
+
+	// Status defines the current state of BackendTLSPolicy.
+	Status PolicyStatus `json:"status,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+// BackendTLSPolicyList contains a list of TCPRoute
+type BackendTLSPolicyList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []BackendTLSPolicy `json:"items"`
+}
+
+// BackendTLSPolicySpec defines the desired state of
+// BackendTLSPolicy.
+// Note: there is no Override or Default policy configuration.
+//
+// Support: Extended
+type BackendTLSPolicySpec struct {
+	// TargetRef identifies an API object to apply policy to.
+	// Services are the only valid API target references.
+	// Note that this config applies to the entire referenced resource
+	// by default, but this default may change in the future to provide
+	// a more granular application of the policy.
+	TargetRef PolicyTargetReference `json:"targetRef"`
+
+	// TLS contains backend TLS policy configuration.
+	TLS *BackendTLSPolicyConfig `json:"tls"`
+}
+
+// BackendTLSPolicyConfig contains backend TLS policy configuration.
+// +kubebuilder:validation:XValidation:message="must not contain both CertRefs and StandardCerts",rule="(has(self.certRefs) && size(self.certRefs) > 0 && has(self.standardCerts) && self.standardCerts != ”)"
+// +kubebuilder:validation:XValidation:message="must specify either CertRefs or StandardCerts",rule="!(has(self.certRefs) && size(self.certRefs) > 0 || has(self.standardCerts) && self.standardCerts != ”)"
+type BackendTLSPolicyConfig struct {
+	// CertRefs contains one or more references to
+	// Kubernetes objects that contain PEM-encoded TLS certificates,
+	// which are used to establish a TLS handshake between the gateway
+	// and backend pod.
```

Comment:
Nit: Generally prefer to have k8s resources capitalized (comment applies throughout)
```suggestion
	// which are used to establish a TLS handshake between the Gateway
	// and backend Pod.
```